File: Code/Components/DanpheEMR.ServerModel/AccountingModels/GroupMappingModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel
{
    public class GroupMappingModel
    {
        [Key]
        public int GroupMappingId { get; set; }
        public string Description { get; set; }
        public int? Section { get; set; }    
        public string Details { get; set; }
        public int? VoucherId { get; set; }
        public string Remarks { get; set; }
        [NotMapped]
        public virtual List<MappingDetailModel> MappingDetail { get; set; }

    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/accounting/transactions/shared/group-mapping.model.ts
import {
    NgForm,
    FormGroup,
    FormControl,
    Validators,
    FormBuilder,
    ReactiveFormsModule
} from '@angular/forms'
import { MappingDetailModel } from "./mapping-detail.model";

export class GroupMappingModel {
    public GroupMappingId: number = 0;
    public Description: string = null;
    public Section: number = 0;   
    public MappingDetail: Array<MappingDetailModel> = new Array<MappingDetailModel>();
    public Details: string = null;
    public VoucherId: number = 0;
    public Remarks: string = null;
}